src/StoreApi/Routes/CartUpdateCustomer.php

```diff
@@ -72,16 +72,20 @@ public function get_args() {
 				'permission_callback' => '__return_true',
 				'args'                => [
 					'billing_address'  => [
-						'description' => __( 'Billing address.', 'woo-gutenberg-products-block' ),
-						'type'        => 'object',
-						'context'     => [ 'view', 'edit' ],
-						'properties'  => $this->billing_address_schema->get_properties(),
+						'description'       => __( 'Billing address.', 'woo-gutenberg-products-block' ),
+						'type'              => 'object',
+						'context'           => [ 'view', 'edit' ],
+						'properties'        => $this->billing_address_schema->get_properties(),
+						'sanitize_callback' => [ $this->billing_address_schema, 'sanitize_callback' ],
+						'validate_callback' => [ $this->billing_address_schema, 'validate_callback' ],
 					],
 					'shipping_address' => [
-						'description' => __( 'Shipping address.', 'woo-gutenberg-products-block' ),
-						'type'        => 'object',
-						'context'     => [ 'view', 'edit' ],
-						'properties'  => $this->shipping_address_schema->get_properties(),
+						'description'       => __( 'Shipping address.', 'woo-gutenberg-products-block' ),
+						'type'              => 'object',
+						'context'           => [ 'view', 'edit' ],
+						'properties'        => $this->shipping_address_schema->get_properties(),
+						'sanitize_callback' => [ $this->shipping_address_schema, 'sanitize_callback' ],
+						'validate_callback' => [ $this->shipping_address_schema, 'validate_callback' ],
```

Comment:
Even though we're copying billing to shipping in https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/3552/files#diff-c8a67488bba0001a6420427352799810b838b6b9a7f18626427e69e7e47fb317R109 those two lines are still running and causing #3592